### PR TITLE
refactor(ui): hard code the page slogan

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

hard code the page slogan instead of read from SIE. The slogan settings will be removed. 

## Testing

test locally

### CODE REVIEW

File: packages/phrases-ui/src/locales/zh-cn.ts [hunk 1]
```diff
@@ -81,10 +81,12 @@ const translation = {
...
     link_phone: '绑定手机',
     link_email_or_phone: '绑定邮箱或手机号',
-    link_email_description: '绑定邮箱以保障您的账号安全',
-    link_phone_description: '绑定手机号以保障您的账号安全',
-    link_email_or_phone_description: '绑定邮箱或手机号以保障您的账号安全',
-    continue_with_more_information: '为保障您的账号安全，需要您补充以下信息。',
+    link_email_description: '绑定邮箱以保障你的账号安全',
+    link_phone_description: '绑定手机号以保障你的账号安全',
+    link_email_or_phone_description: '绑定邮箱或手机号以保障你的账号安全',
+    continue_with_more_information: '为保障你的账号安全，需要你补充以下信息。',
```
Comment (charIeszhao) [inline packages/phrases-ui/src/locales/zh-cn.ts:87]: nit: 
```suggestion
    continue_with_more_information: '为保障账号安全，需补充以下信息。',
```
Also, why this sentence ends with a period `。` while the others don't?

File: packages/phrases-ui/src/locales/zh-cn.ts [hunk 2]
```diff
@@ -81,10 +81,12 @@ const translation = {
     link_email: '绑定邮箱',
     link_phone: '绑定手机',
     link_email_or_phone: '绑定邮箱或手机号',
-    link_email_description: '绑定邮箱以保障您的账号安全',
-    link_phone_description: '绑定手机号以保障您的账号安全',
-    link_email_or_phone_description: '绑定邮箱或手机号以保障您的账号安全',
-    continue_with_more_information: '为保障您的账号安全，需要您补充以下信息。',
+    link_email_description: '绑定邮箱以保障你的账号安全',
+    link_phone_description: '绑定手机号以保障你的账号安全',
+    link_email_or_phone_description: '绑定邮箱或手机号以保障你的账号安全',
```
Comment (charIeszhao) [inline packages/phrases-ui/src/locales/zh-cn.ts:86]: nit: Drop `你的`
```suggestion
    link_email_description: '绑定邮箱以保障账号安全',
    link_phone_description: '绑定手机号以保障账号安全',
    link_email_or_phone_description: '绑定邮箱或手机号以保障账号安全',
```

File: packages/phrases-ui/src/locales/zh-cn.ts [hunk 3]
```diff
@@ -63,9 +63,9 @@ const translation = {
       'The {{type}} {{value}} is linked to another account. Please try another {{type}}', // UNTRANSLATED
     bind_account_title: '绑定或创建帐号',
     social_create_account: '没有帐号？你可以创建一个帐号并绑定。',
-    social_link_email: 'You can link another email', // UNTRANSLATED,
-    social_link_phone: 'You can link another phone', // UNTRANSLATED,
-    social_link_email_or_phone: 'You can link another email or phone', // UNTRANSLATED,
+    social_link_email: 'You can link another email', // UNTRANSLATED
+    social_link_phone: 'You can link another phone', // UNTRANSLATED
+    social_link_email_or_phone: 'You can link another email or phone', // UNTRANSLATED
```
Comment (charIeszhao) [inline packages/phrases-ui/src/locales/zh-cn.ts:68]:
```suggestion
    social_link_email: '绑定其它邮箱',
    social_link_phone: '绑定其它手机号',
    social_link_email_or_phone: '绑定其它邮箱或手机号',
```

File: packages/phrases-ui/src/locales/zh-cn.ts [hunk 4]
```diff
@@ -81,10 +81,12 @@ const translation = {
...
     link_email_or_phone: '绑定邮箱或手机号',
-    link_email_description: '绑定邮箱以保障您的账号安全',
-    link_phone_description: '绑定手机号以保障您的账号安全',
-    link_email_or_phone_description: '绑定邮箱或手机号以保障您的账号安全',
-    continue_with_more_information: '为保障您的账号安全，需要您补充以下信息。',
+    link_email_description: '绑定邮箱以保障你的账号安全',
+    link_phone_description: '绑定手机号以保障你的账号安全',
+    link_email_or_phone_description: '绑定邮箱或手机号以保障你的账号安全',
+    continue_with_more_information: '为保障你的账号安全，需要你补充以下信息。',
+    create_your_account: '注册你的账号',
```
Comment (charIeszhao) [inline packages/phrases-ui/src/locales/zh-cn.ts:88]:
```suggestion
    create_your_account: '注册新账号',
```

**Review by xiaoyijun — APPROVED** [review]
LGTM

File: packages/phrases-ui/src/locales/zh-cn.ts (hunk 1 above)
Comment (simeng-li) [reply packages/phrases-ui/src/locales/zh-cn.ts:87]: might be legacy issue we have a separate PR to refine all these contents.

File: packages/phrases-ui/src/locales/zh-cn.ts (hunk 2 above)
Comment (simeng-li) [reply packages/phrases-ui/src/locales/zh-cn.ts:86]: will address seperately

File: packages/phrases-ui/src/locales/zh-cn.ts (hunk 4 above)
Comment (simeng-li) [reply packages/phrases-ui/src/locales/zh-cn.ts:88]: will address separately

File: packages/phrases-ui/src/locales/zh-cn.ts (hunk 3 above)
Comment (simeng-li) [reply packages/phrases-ui/src/locales/zh-cn.ts:68]: will address separately

**Pull request merged**